proto/osmosis/superfluid/tx.proto

```diff
@@ -225,4 +229,16 @@ message MsgAddToConcentratedLiquiditySuperfluidPositionResponse {
     (gogoproto.nullable) = false
   ];
   uint64 lock_id = 4 [ (gogoproto.moretags) = "yaml:\"lock_id\"" ];
+}
+
+// ===================== MsgLockExistingFullRangePositionAndSFStake
+message MsgLockExistingFullRangePositionAndSFStake {
+  uint64 position_id = 1 [ (gogoproto.moretags) = "yaml:\"position_id\"" ];
+  string sender = 2 [ (gogoproto.moretags) = "yaml:\"sender\"" ];
```

Comment:
isn't sender always position owner? wondering if we can just use `Position.Address`

Reply:
No actually, this field would get automatically get generated thus can be deleted

Reply:
@mattverse is "sender" in the other protos here also unnecessary?